data/ecosystems/s/starknet.toml

```diff
@@ -52,6 +53,7 @@ github_organizations = [
   "https://github.com/keep-starknet-strange",
   "https://github.com/kkrt-labs",
   "https://github.com/OnchainBlock",
+  "https://github.com/SphinxDB",
```

Comment:
This is covered in your sphinx protocol ecosystem, so don't need to add it here.

Reply:
Okay fixed